controllers/productListController.js

```diff
@@ -0,0 +1,12 @@
+const productListService = require('../services/productListService');
+
+const productList = async (req, res) => {
```

Comment:
```suggestion
const getProductList = async (req, res) => {
```

- 함수명은 동사형으로 시작하여, 어떤 기능을 '수행하는지'를 명확히 하는 것이 좋습니다 :)